fe/fe-core/src/main/java/com/starrocks/sql/common/MetaUtils.java

```diff
@@ -78,6 +78,35 @@ public static Table getTable(long dbId, long tableId) {
         return table;
     }
 
+    public static Table getTableByUUID(String tableUUID) {
+        String[] splits = tableUUID.split("\\.");
+        if (splits.length < 3) {
```

Comment:
When tableUUID splits length less than 3?

Reply:
when it's native table, the tableUUID is table id.